### DISCUSSION TITLE
Does this work with supabase ui Auth component?

### DISCUSSION THREAD

**Discussion by vedantroy** [mitchelvanbever/remix-auth-supabase#25] [Q&A] [answered]
Hi! Thanks for making this library.
I was wondering if this works with the Supabase UI Auth component?
https://github.com/supabase/ui

**Comment by rphlmr** [comment, marked as answer]
Hi,
Sorry it's not tailored to work with it :/ 
Supabase UI > Auth is client side authentication while remix-auth-supabase is server side auth ;)
